cli/main.go

```diff
@@ -43,11 +43,10 @@ func buildVersionString() string {
 			fmt.Fprintf(versionsWriter, "\t%s\n", image)
 		}
 	}
-	fmt.Fprint(versionsWriter, "\n")
-	fmt.Fprintf(versionsWriter, "reference values for %s platform:\n", platforms.AKSCloudHypervisorSNP.String())
-	fmt.Fprintf(versionsWriter, "\truntime handler:\tcontrast-cc-%s\n", manifest.TrustedMeasurement[:32])
-	fmt.Fprintf(versionsWriter, "\tlaunch digest:\t%s\n", manifest.TrustedMeasurement)
-	fmt.Fprintf(versionsWriter, "\tgenpolicy version:\t%s\n", constants.GenpolicyVersion)
+	if refValues, err := json.MarshalIndent(manifest.EmbeddedReferenceValues(), "\t", "  "); err == nil {
+		fmt.Fprintf(versionsWriter, "embedded reference values:\t%s\n", refValues)
```

Comment:
This reduced readability of the output a lot. If we want to keep this  as json output, we should move it to the end and start the json doc in a separate line so you can cut it and process it with jq if needed.

Reply:
I'd rather have a `--json` for this use case.

Reply:
That would mean a refactoring of the command structure, as we are currently using the builtin --version flag, not a command,  right?